package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "lichtblick",
-  "version": "1.2.1",
+  "name": "foxbox",
```

Comment:
please keep the renaming for lichtblick

Reply:
Done! It was replaced when I was solving conflicts and didn't notice it has returned to foxbox. good catch.